SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DMDCollectionViewDataSource.swift

```diff
@@ -102,8 +103,7 @@ extension DonationsMoreDetailsController: UICollectionViewDataSource {
                 withReuseIdentifier: DMDPhotoSupplementaryView.reuseIdentifier,
                 for: indexPath) as? DMDPhotoSupplementaryView {
                 
-                let person = donation.person
-                person.flatMap { photoView.configure($0) }
+                photoView.configure(withURL: donation.bannerImagePath ?? "")
```

Comment:
if passing `""`, could it be nil instead? nil is standard in swift for "no value"

Reply:
Great one! Already covered in the lasted commit.